test/snapp_auction.js

```diff
@@ -89,13 +90,13 @@ contract("SnappAuction", async (accounts) => {
       await setupEnvironment(MintableERC20, instance, token_owner, [user_1], 2)
       await instance.placeSellOrder(1, 2, 1, 1, { from: user_1 })
 
-      await waitForNBlocks(21, owner)
+      await waitForNSeconds(601, owner)
```

Comment:
Is there any reason you didn't use 60*3 + 1  for 3 minutes and 1 second?

Reply:
Great catch! Who could have known that 3 minutes is not 600 seconds.

🎣